x/payment/client/testutil/integration_test.go

```diff
@@ -40,10 +40,6 @@ func NewIntegrationTestSuite(cfg cosmosnet.Config) *IntegrationTestSuite {
 func (s *IntegrationTestSuite) SetupSuite() {
 	s.T().Log("setting up integration test suite")
 
-	if testing.Short() {
-		s.T().Skip("skipping test in unit-tests mode.")
-	}
-
```

Comment:
Isn't skipping somewhere still necessary?

Reply:
not for this test, as its small enough for the detector to handle